src/js/createCase.js

```diff
@@ -59,8 +59,7 @@ export async function createSupportCase(userInfo, fields) {
       },
       sessionDetails: {
         createdBy: `${userInfo.user.username}`,
-        environment: `${window.insights.chrome.isBeta() ? 'Production Beta' : 'Production'}`,
-        description: `${currentHash}`,
+        environment: `${window.insights.chrome.isBeta() ? 'Production Beta' : 'Production'}, ${currentHash}`,
```

Comment:
No need to repeat `Production` twice
```suggestion
        environment: `Production${window.insights.chrome.isBeta() ? ' Beta' : ''}, ${currentHash}`,
```